frontend/src/main/scala/bloop/dap/DebuggeeRunner.scala

```diff
@@ -40,14 +40,17 @@ private final class MainClassDebugAdapter(
 
   def run(debugLogger: DebugSessionLogger): Task[ExitStatus] = {
     val workingDir = state.commonOptions.workingPath
+    // Metals used to add the `-J` prefix but it is not needed anymore
+    // So we cautiously strip it off
```

Comment:
Could we add TODO with an issue to follow up? I think once we release Metals with the next Bloop version we should be good to remove it.  Created the issue here: https://github.com/scalacenter/bloop/issues/1456 

Could you also add it to any other place that we might want to remove the workaround from?

Reply:
> Could you also add it to any other place that we might want to remove the workaround from?
What other places do you have in mind?

Reply:
I think the other places you are mentioning are not workarounds. In their cases the options are coming from the Bloop command so the `-J` prefix is expected.